### PR TITLE
Improvements infrastructure

### PR DESCRIPTION
**Pull request by braybaut**
## Description
* Remove the AWS profile definition, this required having an AWS profile called **default**  and doesn't possible uses another profile.
* Unifies all local values in the same local block 
* Modify readme with additional terraform information
* Add a simple test code to test the infrastructure 
  
 ## How Has This Been Tested?
 Running  `go test ` inside test/ directory
 
 ## Types of changes
 * [x]  Bug fix (non-breaking change which fixes an issue)
 * [ ]  New feature (non-breaking change which adds functionality)
 * [ ]  Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by senior7515** [comment]
cc: @ivotron

File: aws/readme.md [hunk 1]
```diff
@@ -7,16 +7,66 @@ After completing these steps, please follow the required steps in the [project r
...
+| Name | Description |
+|------|-------------|
+| prometheus | n/a |
+| public\_key\_path | n/a |
+| redpanda | n/a |
+| ssh\_user | n/a |
+
+## Test
+
+Test the infrastructura with the following command:
```
Comment (0xdiba) [inline aws/readme.md:69]: `infrastructura` -> `infrastructure`

File: aws/readme.md [hunk 2]
```diff
@@ -7,16 +7,66 @@ After completing these steps, please follow the required steps in the [project r
...
+
+## Requirements
+
+| Name | Version |
+|------|---------|
+| aws | 3.29.1 |
+| local | 2.1.0 |
+| random | 3.1.0 |
+
+## Providers
```
Comment (0xdiba) [inline aws/readme.md:22]: Should we merge requirements and providers?

File: aws/readme.md [hunk 3]
```diff
@@ -7,16 +7,66 @@ After completing these steps, please follow the required steps in the [project r
...
+|------|-------------|------|---------|:--------:|
+| aws\_region | The AWS region to deploy the infrastructure on | `string` | `"us-west-2"` | no |
+| distro | The default distribution to base the cluster on | `string` | `"ubuntu-focal"` | no |
+| distro\_ami | n/a | `map(string)` | <pre>{<br>  "amazon-linux-2": "ami-01ce4793a2f45922e",<br>  "debian-buster": "ami-0f7939d313699273c",<br>  "debian-stretch": "ami-072ad3956e05c814c",<br>  "fedora-31": "ami-0e82cc6ce8f393d4b",<br>  "fedora-32": "ami-020405ee5d5747724",<br>  "rhel-8": "ami-087c2c50437d0b80d",<br>  "ubuntu-bionic": "ami-0c1ab2d66f996cd4b",<br>  "ubuntu-focal": "ami-02c45ea799467b51b"<br>}</pre> | no |
+| distro\_ssh\_user | The default user used by the AWS AMIs | `map(string)` | <pre>{<br>  "amazon-linux-2": "ec2-user",<br>  "debian-buster": "admin",<br>  "debian-stretch": "admin",<br>  "fedora-31": "fedora",<br>  "fedora-32": "fedora",<br>  "rhel-8": "ec2-user",<br>  "ubuntu-bionic": "ubuntu",<br>  "ubuntu-focal": "ubuntu"<br>}</pre> | no |
+| enable\_monitoring | Setup a prometheus/grafana instance | `bool` | `true` | no |
+| instance\_type | Default redpanda instance type to create | `string` | `"i3.2xlarge"` | no |
```
Comment (0xdiba) [inline aws/readme.md:53]: `Instance type used for redpanda nodes`?

File: aws/readme.md [hunk 4]
```diff
@@ -7,16 +7,66 @@ After completing these steps, please follow the required steps in the [project r
...
+
+| Name | Description |
+|------|-------------|
+| prometheus | n/a |
+| public\_key\_path | n/a |
+| redpanda | n/a |
+| ssh\_user | n/a |
+
+## Test
+
```
Comment (0xdiba) [inline aws/readme.md:68]: Let's make it explicit that a go installation is needed to run the tests.

**Review by 0xdiba — COMMENTED** [review]
Great work :rocket: Thanks for doing this.
Only some minor comments.
Could you also separate the changes into 3 commits? (readme changes, terraform code changes, and testing code)

**Review by 0xdiba — COMMENTED** [review]
Also, please sign the CLA :fountain_pen: :)

**Comment by senior7515** [comment]
@braybaut @Brayan-Bautista-Globant  - i think the cla assistant check is still failing.

File: aws/readme.md (hunk 1 above)
Comment (braybaut) [reply aws/readme.md]: Solved

File: aws/readme.md (hunk 2 above)
Comment (braybaut) [reply aws/readme.md:22]: Defining the versions supported  we ensure that the code will work with those versions

File: aws/readme.md (hunk 3 above)
Comment (braybaut) [reply aws/readme.md:53]: Yes, the same values that variables.tf

**Comment by braybaut** [comment]
> @braybaut @Brayan-Bautista-Globant - i think the cla assistant check is still failing.

Maybe because I signed the CLA :/

**Comment by braybaut** [comment]
> Also, please sign the CLA fountain_pen :)

I did it, maybe the CLA check is failing

**Comment by ivotron** [comment]
> > Also, please sign the CLA fountain_pen :)
> 
> I did it, maybe the CLA check is failing

sorry about that @braybaut. Can you please click on the "recheck it" link shown in the message from the CLA bot? Pasting it here for convenience: https://cla-assistant.io/check/vectorizedio/deployment-automation?pullRequest=23

Also, I noticed you're using two github IDs, `@Brayan-Bautista-Globant` for your commits, and `@braybaut` for the repo from where this PR was originating from. This might be the underlying issue. One way this could be fixes is by rebase interactively in order to change the author id of commits (see how to do this [here](https://www.deployhq.com/git/faqs/update-author-committer-multiple-git-commits))

**Comment by braybaut** [comment]
> > > Also, please sign the CLA fountain_pen :)
> > 
> > 
> > I did it, maybe the CLA check is failing
> 
> sorry about that @braybaut. Can you please click on the "recheck it" link shown in the message from the CLA bot? Pasting it here for convenience: https://cla-assistant.io/check/vectorizedio/deployment-automation?pullRequest=23
> 
> Also, I noticed you're using two github IDs, `@Brayan-Bautista-Globant` for your commits, and `@braybaut` for the repo from where this PR was originating from. This might be the underlying issue. One way this could be fixes is by rebase interactively in order to change the author id of commits (see how to do this [here](https://www.deployhq.com/git/faqs/update-author-committer-multiple-git-commits))

Yes, thank I removed the other Github ID and now the CLA is signed

File: aws/readme.md (hunk 2 above)
Comment (0xdiba) [reply aws/readme.md:22]: Yeah I agree with that but is there a reason to track it both under requirements and providers?

Comment (braybaut) [reply aws/readme.md:22]: Maybe not, currently, the code is very clean and just deployy some instances, in the future if the code depends on a specific feature, we can contemplate merging providers and requirements. I could remove providers and requirements, what do you think?

Comment (0xdiba) [reply aws/readme.md:22]: I'm fine with keeping on or the other since they point to exactly the same versions :+1: .

**Review by 0xdiba — APPROVED** [review]

**Review by rkruze — APPROVED** [review]
This looks good to me, we should probably format the gcp readme the same way.

**Comment by senior7515** [comment]
@braybaut thanks so much for this! & congratulations.

**Pull request merged**